.github/workflows/validate-pr.yml

```
@@ -258,7 +258,7 @@ jobs:
     - name: Setup GraalVM environment
       uses: olafurpg/setup-scala@v10
       with:
-        java-version: graalvm@19.0.0
+        java-version: graalvm@20.3.0
```

Should be fine to update back to v21 now
```suggestion
        java-version: graalvm@21.0.0
```